test/sqlite/sqllogic_command.cpp

```diff
@@ -88,8 +89,16 @@ unique_ptr<MaterializedQueryResult> Command::ExecuteQuery(ExecuteContext &contex
 	if (TestForceReload() && TestForceStorage()) {
 		RestartDatabase(context, connection, context.sql_query);
 	}
-
-	return connection->Query(context.sql_query);
+	// return connection->Query(context.sql_query);
+	auto ccontext = connection->context;
+	auto result = ccontext->Query(context.sql_query, true);
```

Comment:
This should probably be reverted or made into a separate `unittest` flag?
With a corresponding CI run that checks everything succeeds with this flag set?

Reply:
Maybe `#ifdef DUCKDB_ALTERNATIVE_VERIFY`